CliWrap/Cli.cs

```diff
@@ -255,33 +271,23 @@ public void ExecuteAndForget()
         /// <summary>
         /// Executes CLI with given input, waits until completion asynchronously and returns output
         /// </summary>
-        public Task<ExecutionOutput> ExecuteAsync(ExecutionInput input)
-            => ExecuteAsync(input, CancellationToken.None);
-
-        /// <summary>
-        /// Executes CLI with given input, waits until completion asynchronously and returns output
-        /// </summary>
-        public Task<ExecutionOutput> ExecuteAsync(string arguments, CancellationToken cancellationToken)
-            => ExecuteAsync(new ExecutionInput(arguments), cancellationToken);
-
-        /// <summary>
-        /// Executes CLI with given input, waits until completion asynchronously and returns output
-        /// </summary>
-        public Task<ExecutionOutput> ExecuteAsync(string arguments)
-            => ExecuteAsync(new ExecutionInput(arguments), CancellationToken.None);
+        public Task<ExecutionOutput> ExecuteAsync(string arguments, CancellationToken cancellationToken = default(CancellationToken), IStandardBufferHandler standardBufferHandler = null)
+            => ExecuteAsync(new ExecutionInput(arguments), cancellationToken, standardBufferHandler);
 
         /// <summary>
         /// Executes CLI without input, waits until completion asynchronously and returns output
         /// </summary>
-        public Task<ExecutionOutput> ExecuteAsync(CancellationToken cancellationToken)
-            => ExecuteAsync(ExecutionInput.Empty, cancellationToken);
+        public Task<ExecutionOutput> ExecuteAsync(CancellationToken cancellationToken = default(CancellationToken), IStandardBufferHandler standardBufferHandler = null)
+            => ExecuteAsync(ExecutionInput.Empty, cancellationToken, standardBufferHandler); 
 
         /// <summary>
         /// Executes CLI without input, waits until completion asynchronously and returns output
         /// </summary>
         public Task<ExecutionOutput> ExecuteAsync()
```

Comment:
You removed the argument-less Execute but kept one for ExecuteAsync. I suggest to remove this one too to make it consistent.